bin/p2-balancer/main.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/hashicorp/consul/api"
+	"github.com/square/p2/pkg/balancer"
+	"github.com/square/p2/pkg/config"
+	"github.com/square/p2/pkg/logging"
+)
+
+// serpro (service proxy) is a simple TCP proxy for services registered
+// in the Consul registry.
+
+func main() {
+	cfg, err := config.LoadFromEnvironment()
```

Comment:
Unrelated to this PR, but I feel like the API for config should be LoadFromEnvironment(root string, some_struct interface{})

It will yaml marshal into the struct pointer you pass in. Then you don't need all these ReadString and ReadMap etc